index.html

```diff
@@ -0,0 +1,165 @@
+            <p class="para2">@Kenyatta University Conference center, Chandaria innovation centre</p>
+        </div>
+    </section>
+    <section id="about">
+        <div class="header">
+            <h2>Main Program</h2>
+            <div class="borderbottom"></div>
+        </div>
+        <div class="main-program-container">
+            <div class="activity">
+                <img src="images/drought.png" alt="" sizes="" srcset="">
+                <h4>Lecture</h4>
+                <p>The 3-day meeting will incorporate poster exhibitions, presentations and interactive sessions from participants.</p>
+            </div>
+            <div class="activity">
+                <img src="images/Fish-icon.png" alt="" sizes="" srcset="">
+                <h4>Lecture</h4>
+                <p>The 3-day meeting will incorporate poster exhibitions, presentations and interactive sessions from participants.</p>
+            </div>
+            <div class="activity">
+                <img src="images/molecular.png" alt="" sizes="" srcset="">
+                <h4>Lecture</h4>
+                <p>The 3-day meeting will incorporate poster exhibitions, presentations and interactive sessions from participants.</p>
+            </div>
+            <div class="activity">
+                <img src="images/objective.png" alt="" sizes="" srcset="">
+                <h4>Lecture</h4>
+                <p>The 3-day meeting will incorporate poster exhibitions, presentations and interactive sessions from participants.</p>
+            </div>
+            <div class="activity">
+                <img src="images/h2o.png" alt="" sizes="" srcset="">
+                <h4>Lecture</h4>
+                <p>The 3-day meeting will incorporate poster exhibitions, presentations and interactive sessions from participants.</p>
+            </div>
+        </div>
+        
+        <div class="buttondiv">
+            <button>Join Climate Change Conference</button>
+        </div>
+        <div class="see-whole-program">
+            <a href="http://">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+    <section id="featured-speakers">
+        <h2>Featured Speakers</h2>
+        <div id="speaker-mother-container">
+
+        </div>
+       
+        
+        
+    </section>
+    <div class="see-more-btn"><button value="More">More<img src="images/rename.png" alt="dropdown"></button></div>
+
+    <div class="section4" id="desktop-partners">
+        <div><h2>Partners</h2></div>
+        <div class="sponsor-container ">
+            <img class="sponsor-logo" src="images/Gazprom.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/freedom.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/Besthomes.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/loremipsum.jpg" alt="sponsor-logo">
+            <img class="sponsor-logo" src="images/wildlife.jpg" alt="sponsor-logo">
+        </div>
+    </div>
+
```

Comment:
- [ ] Hello @collins-kiprotich, you did a great job implementing the changes requested by the previous reviewer 👏. However, I have noticed that the partner's logos are missing in the `partners section` please would you like to add them back, because you had them previously? 👌
### Now 👇️
![Screenshot from 2022-11-24 20-54-11](https://user-images.githubusercontent.com/89228743/203862635-42963599-254f-4a12-a59c-e33b7d1a619b.png)

### Before 👇️
![Screenshot from 2022-11-24 21-03-42](https://user-images.githubusercontent.com/89228743/203862652-f3718876-7e27-4809-9e62-5157281a31a5.png)

Reply:
Thanks @ridwanediallo for the feedback. Regarding the requested change on my partners section, the logos are showing when I run it on my local machine, though it takes little longer for them to load to the page.